### PR TITLE
Staging to master for making release

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Description

After https://github.com/microsoft/recommenders/pull/1602 was fixed, merging to master for new release

### Checklist:

- [ ] I have followed the [contribution guidelines](../CONTRIBUTING.md) and code style for this project.
- [ ] I have added tests covering my contributions.
- [ ] I have updated the documentation accordingly.
- [ ] This PR is being made to `staging branch` and not to `main branch`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,41 +37,51 @@ and currently does not support version 3.8 and above. It is recommended to insta
...
+    1. Ensure software required for compilation and Python libraries
+       is installed.
+
+       + On Linux this can be supported by adding:
+
+         ```bash
+         sudo apt-get install -y build-essential libpython<version>
+         ``` 
+
+         where `<version>` should be `3.6` or `3.7` as appropriate.
```
Comment (simonzhaoms) [inline README.md:51]: Dose it support Python 3.8 now? To support Databricks 7.3+, Python 3.8 is required.

File: SETUP.md [hunk 2]
```diff
@@ -397,7 +397,7 @@ You can then open the Jupyter notebook server at http://localhost:8888
 
 The process of making a new release and publishing it to pypi is as follows:
 
-First make sure that the tag that you want to add, e.g. `0.6.0`, is added in [recommenders.py/__init__.py](recommenders.py/__init__.py). Follow the [contribution guideline](CONTRIBUTING.md) to add the change.
+First make sure that the tag that you want to add, e.g. `0.6.0`, is added in [recommenders.py/`__init__.py`](recommenders.py/__init__.py). Follow the [contribution guideline](CONTRIBUTING.md) to add the change.
```
Comment (simonzhaoms) [inline SETUP.md:400]:
```
[recommenders.py/`__init__.py`](recommenders.py/__init__.py)
```

should be

```
[`recommenders/__init__.py`](recommenders/__init__.py)
```

File: README.md (hunk 1 above)
Comment (simonzhaoms) [reply README.md:51]: 3.8 may also need to be added to:
https://github.com/microsoft/recommenders/blob/f4636b2e8e74d87233fd53958c5e0526674fc7f6/setup.py#L127-L134

Comment (miguelgfierro) [reply README.md:51]: that's a good point, if my memory doesn't failed me, we had some problems with 3.8 and numpy? @anargyri might know more when he is back

File: SETUP.md (hunk 2 above)
Comment (miguelgfierro) [reply SETUP.md:400]: fixed in https://github.com/microsoft/recommenders/pull/1606/commits/48aab4792dee0c9871d2d141122b0f45f4b34453

File: README.md (hunk 1 above)
Comment (anargyri) [reply README.md:51]: The unit tests work with Python 3.8. See https://github.com/microsoft/recommenders/pull/1617

**Pull request merged**